WordPress/Classes/ViewRelated/Post/PrivateSiteURLProtocol.m

```diff
@@ -92,34 +95,50 @@ - (void)startLoading
 {
     NSMutableURLRequest *mRequest = [self.request mutableCopy];
     [mRequest addValue:[NSString stringWithFormat:@"Bearer %@", [[self class] bearerToken]] forHTTPHeaderField:@"Authorization"];
-    self.connection = [NSURLConnection connectionWithRequest:mRequest delegate:self];
+    NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration defaultSessionConfiguration];
+    self.session = [NSURLSession sessionWithConfiguration:sessionConfiguration delegate:self delegateQueue:nil];
+    self.sessionTask = [self.session dataTaskWithRequest:mRequest];
+    [self.sessionTask resume];
 }
 
 - (void)stopLoading
 {
-    [self.connection cancel];
+    [self.sessionTask cancel];
+    [self.session invalidateAndCancel];
+    self.sessionTask = nil
+    self.session = nil
```

Comment:
Swift thinking :(

Sent from my iPhone

> On 3 May 2016, at 15:44, Eric <notifications@github.com> wrote:
> 
> In WordPress/Classes/ViewRelated/Post/PrivateSiteURLProtocol.m:
> 
> >  }
> >  
> >  - (void)stopLoading
> >  {
> > -    [self.connection cancel];
> > +    [self.sessionTask cancel];
> > +    [self.session invalidateAndCancel];
> > +    self.sessionTask = nil
> > +    self.session = nil
> Missing ;s
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>